yb-voyager/cmd/root.go

```diff
@@ -77,7 +80,7 @@ func shouldLock(cmd *cobra.Command) bool {
 	if cmd.Use == "initiate" && cmd.Parent().Use == "cutover" {
 		return false
 	}
-	if cmd.HasParent() && cmd.Parent().Use == "fall-forward" {
+	if cmd.Use == "switchover" && cmd.Parent().Use == "fall-forward" {
 		return false
```

Comment:
Suggestion: should we use `cmd.CommandPath()` (https://pkg.go.dev/github.com/spf13/cobra#Command.CommandPath) to get the full command name and explicitly use them in comparisons. For example, we can have a list of commands for which lock is not needed. Then we can just check if the CommandPath() exists in the list.
```
func shouldLock(cmd ...) bool {
  noLockNeededList := []string{
    "yb-voyager version",
    "yb-voyager import data status",
    "yb-voyager export data status",
    "yb-voyager cutover initiate",
    "yb-voyager fall-forward switchover",
  }
  return !slice.Contains(noLockNeededList, cmd.CommandPath())
}
```